lib/tools/emacs/erlang.el

```diff
@@ -2111,14 +2121,10 @@ provides the same functionality as erlang-man-function except for
 that it does not ask the user to confirm the function name before
 opening the man page for the function."
   (interactive)
-  (progn
-    (erlang-man-function (erlang-default-function-or-module))
-    (sleep-for 0 800) ; A hack to make sure that the function scrolls
-                      ; to the description of the function when it is
-                      ; the first time that the man page for a module
-                      ; is opened
-    (erlang-man-function (erlang-default-function-or-module))
-    ))
+  (let ((name (erlang-default-function-or-module)))
+    (if name
+        (erlang-man-function name)
+      (error "No function name under the cursor. Place the cursor over the function name you want to find the man page for and try again"))))
```

Comment:
Too long line.  Second sentence could be removed as far as i am concerned.

Reply:
Thanks for the review! I have removed the second sentence in commit 
7864c56.

Reply:
Ok, no problem.  Thanks for the push.